WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -282,7 +282,7 @@ - (void)synchPostFormatsAndDo:(void(^)(void))completionBlock
 - (void)addPostPropertiesObserver
 {
     [self.post addObserver:self
-             forKeyPath:@"post_thumbnail"
+             forKeyPath:@"featuredImage"
```

Comment:
Can we use a constant for this keypath, as we refer to it in quite a few places? :)